pkg/setting/setting.go

```diff
@@ -1450,6 +1461,68 @@ func readAlertingSettings(iniFile *ini.File) error {
 	return nil
 }
 
+func readGRPCServerSettings(cfg *Cfg, iniFile *ini.File) error {
```

Comment:
I assume we would add the `grpc_server` section to `defaults.ini` only after we promote the feature to beta, right?

Reply:
@ryantxu thoughts? @ArturWierzbicki and i were thinking we'd hold off on this for now

Reply:
We should not add it to defaults (or docs) until it is in beta -- but I think we still need to be able to use the settings.

So I think this is good as is